docs/source/markdown/podman-image-scp.1.md

```diff
@@ -28,6 +28,14 @@ Suppress the output
 
 Print usage statement
 
+#### **--rootless**
+
+Rootless determines whether we are loading locally from root storage to rootless storage. This flag must be used in conjunction with --user.
```

Comment:
loading what?  This doesn't really tell the user what this accomplishes.  How's about:

```suggestion
The rootless option allows a user to transfer images from the root storage to the user's 'rootles' storage.  This option must be used in conjunction with the --user option.
```